specification/resources/apps/models/app.yml

```
@@ -67,6 +67,10 @@ properties:
     title: Time of the app's last configuration update
     type: string
     example: 2020-12-01T00:42:16Z
+  pinned_deployment:
+    allOf:
+      - description: The deployment that the app is pinned to
```

nit: -- I tried to look around and it seems like we end with a period for other `description` fields
```suggestion
      - description: The deployment that the app is pinned to.
```